tests/test_nonci.py

```diff
@@ -5,39 +5,63 @@
 config = """
 from _universum.configuration_support import Variations
 
-configs = Variations([dict(name="test_step", artifacts="test_nonci.txt",
-                           command=["bash", "-c", '''echo "test nonci" > test_nonci.txt'''])])
+configs = Variations([dict(name="artifact check",
+                           command=["bash", "-c", '''cat /artifacts/test_nonci.txt''']),
+#                                                    ^ this helps to check artifact is deleted before launch 
+
+                      dict(name="test_step", artifacts="test_nonci.txt",
+                           command=["bash", "-c", '''echo "pwd:[$(pwd)]" && echo "test nonci" > test_nonci.txt'''])])
+#                                                    ^ this helps to check the path is not changed
 """
 
 
 def test_launcher_output(universum_runner_nonci):
+    """
+    This test verifies that nonci mode changes the behavior of the universum by checking its output to console and log
+    files. Specifically, it checks the following features of the nonci mode:
+     - default output of the step is console, even when running in the terminal
+     - sources are not copied to temp directory
+     - artifacts are deleted before launching configs
+     - version control and review system are not used
+    """
     file_output_expected = "Adding file /artifacts/test_step_log.txt to artifacts"
-    step_log_expected = """/bin/bash -c echo "test nonci" > test_nonci.txt"""
+    pwd_string_in_logs = "pwd:[" + universum_runner_nonci.local.root_directory.strpath + "]"
+
+    universum_runner_nonci.environment.assert_successful_execution(
+        "bash -c 'mkdir /artifacts; echo \"Old artifact\" > /artifacts/test_nonci.txt'")
 
-    console_out_log = universum_runner_nonci.run(config) # defult -lo is console
-    assert file_output_expected not in console_out_log
-    assert step_log_expected in console_out_log
-    # nonci doesn't required to clean artifacts between calls
+    console_out_log = universum_runner_nonci.run(config)
+    assert "Cleaning artifacts..." in console_out_log           # nonci cleans artifacts on project launch
```

Comment:
Please divide this block on two: check "lines expected in logs" and "shall not be in logs"
In my opinion it will be more readable.
```
assert line in log
assert line in log

assert line not in log
...
```

Reply:
done